plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/LocalProjectImpl.java

```diff
@@ -218,6 +223,21 @@ public int visit(Bucket.Entry entry) {
         return result;
     }
 
+    public void hideMetadataFolder() {
```

Comment:
Is it really required to be a part of public API? Can't we do that I the `ensureOpen` method only?